src/core/stream.c

```diff
@@ -393,12 +393,6 @@ QuicStreamClose(
     if (!Stream->Flags.ShutdownComplete) {
 
         if (Stream->Flags.Started && !Stream->Flags.HandleShutdown) {
-            //
```

Comment:
Maybe remove the TODO but keep the comment as a signpost for investigating this scenario?

Reply:
The comment doesn't reflect what the code does or what Nick says the behavior is supposed to be (closing the stream without aborting it is supposed to abort the stream implicitly with error code `0`).
IMO, this comment has a negative value, it is misleading + doesn't point to something that actually needs to be done.